src/timeAgo.README.md

```diff
@@ -0,0 +1,37 @@
+
+# `timeAgo`
+
+Takes an absolute date in [epoch format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description) and returns a string representing relative time ago.
+
+## Usage
+`timeAgo(epoch, opts)`
+
+Returns: `string | false`
+
+Converts an absolute epoch to a relative time ago string
+
+`epoch`
+
+Type: `number`
+
+The date when an event happened in epoch format
+
+`opts`
+
+Type:
+```typescript
+type Opts = {
+    verbose?: boolean, // Return a longer, more descriptive string when true
+    daysUntilAbsolute?: number,  // The cutoff for when dates are returned in absolute format
```

Comment:
should we add something above explaining that this can happen?

Reply:
Explaining what can happen? (Sorry, sometimes the highlight feature in Github, whilst great, can be misleading)